pkg/kanctl/repositoryserver.go

```diff
@@ -0,0 +1,233 @@
+// Copyright 2023 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kanctl
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+)
+
+const (
+	tlsSecretFlag                       = "tlsSecret"
+	repoServerUserFlag                  = "repositoryServerUser"
+	repoServerUserAccessSecretFlag      = "repositoryServerUserAccessSecret"
+	repoServerAdminUserAccessSecretFlag = "repositoryServerAdminUserAccessSecret"
+	repoPasswordSecretFlag              = "repositoryPasswordSecret"
+	repoUserFlag                        = "repositoryUser"
+	locationCredsSecretFlag             = "locationCredsSecret"
+	locationSecretFlag                  = "locationSecret"
```

Comment:
```suggestion
	tlsSecretFlag                       = "tls-secret"
	repoServerUserFlag                  = "repository-server-user"
	repoServerUserAccessSecretFlag      = "repository-server-user-access-secret"
	repoServerAdminUserAccessSecretFlag = "repository-server-admin-user-access-secret"
	repoPasswordSecretFlag              = "repository-password-secret"
	repoUserFlag                        = "repository-user"
	locationCredsSecretFlag             = "location-creds-secret"
	locationSecretFlag                  = "location-secret"
```
We can have them as hiphen separated instead of camel case to be in sync with other commands.